az-core/src/main/java/azkaban/Constants.java

```diff
@@ -167,6 +167,15 @@
 
     // Job property that enables/disables using Kafka logging of user job logs
     public static final String AZKABAN_JOB_LOGGING_KAFKA_ENABLE = "azkaban.job.logging.kafka.enable";
+
+    /*
+     * this parameter is used to replace EXTRA_HCAT_LOCATION that could fail when one of the uris is not available.
+     * EXTRA_HCAT_CLUSTERS has the following format:
+     * other_hcat_clusters = "thrift://hcat1:port,thrift://hcat2:port;thrift://hcat3:port,thrift://hcat4:port"
+     * Each string in the parenthesis is regarded as a "cluster", and we will get a delegation token from each cluster.
+     * The uris(hcat servers) in a "cluster" ensures HA is provided.
+     **/
+    public static final String EXTRA_HCAT_CLUSTERS = "other_hcat_clusters";
```

Comment:
Should this property be put under `JobProperties` class. Also, you might want to use `azkaban.job.other_hcat_clusters`

Reply:
Or, is `other_hcat_cluster`s an existing configuration now?

Reply:
How about azkaban.job.hive.other_hcat_clusters?
Please move the existing config to this class too while you are refactoring.